src/openms/source/ANALYSIS/ID/SiriusAdapterAlgorithm.cpp

```diff
@@ -33,11 +33,16 @@
 // --------------------------------------------------------------------------
 
 #include <OpenMS/ANALYSIS/ID/SiriusAdapterAlgorithm.h>
-
+#include <OpenMS/FORMAT/FeatureXMLFile.h>
```

Comment:
optional (knowing that Im getting picky): sorting includes lexicographically has proven to be better maintainable (avoids duplicate includes -- which are not a performance problem, but noise nevertheless)

Reply:
No worries! At least something to learn here ;)